### PR TITLE
CImg: update to 3.3.0

### PR DESCRIPTION
**Pull request by barracuda156**
#### Description

Update

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.6.8
Xcode 3.2.6

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [ ] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

**Comment by barracuda156** [comment]
CI failed on fetch after a rebase which does not touch the port itself. Apparently their website has a poor connection.

**Comment by barracuda156** [comment]
@reneeotten @herbygillot Could you help with merging this?

File: devel/CImg/Portfile [hunk 1]
```diff
@@ -3,27 +3,27 @@
...
+license             {CeCILL-C CeCILL}
 
 description         C++ Template Image Processing Toolkit
 
 long_description    Open source, C++ toolkit for image processing
 
-homepage            http://cimg.eu \
+homepage            https://cimg.eu \
                     https://github.com/GreycLab/CImg
 master_sites        http://cimg.eu/files/
```
Comment (herbygillot) [inline devel/CImg/Portfile:19]: HTTPS here too plz

Comment (barracuda156) [reply devel/CImg/Portfile:19]: I think I tried to use it, and it failed to fetch. Will check again.

Comment (barracuda156) [reply devel/CImg/Portfile:19]: No, it works fine. I will change it to `https` now.

Comment (barracuda156) [reply devel/CImg/Portfile]: @herbygillot Fixed and rebased to master.

**Pull request merged**